colibre/config.yml

```diff
@@ -208,6 +208,11 @@ scripts:
     title: Step size vs deadtime fraction
     section: Run Performance
     output_file: stepsize_deadtime.png
+  - filename: scripts/deadtime_evolution.py
+    caption: Evolution of the dead time fraction as a function of scale factor.
```

Comment:
Maybe indicate in the caption what the dashed line stands for? (i.e. the mean dead time for the whole run)